recipe/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "python-ffmpeg" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```

```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```
Redundant double quotes `"`